application/F3DStarter.cxx

```diff
@@ -901,18 +905,32 @@ int F3DStarter::Start(int argc, char** argv)
         }
       });
 
-    interactor.addBinding("Left", f3d::interactor::ModifierKeys::NONE, "load_previous_file_group");
-    interactor.addBinding("Right", f3d::interactor::ModifierKeys::NONE, "load_next_file_group");
-    interactor.addBinding("Up", f3d::interactor::ModifierKeys::NONE, "reload_current_file_group");
-    interactor.addBinding("Down", f3d::interactor::ModifierKeys::NONE, "add_current_directories");
-    interactor.addBinding("F11", f3d::interactor::ModifierKeys::NONE, "take_minimal_screenshot");
-    interactor.addBinding("F12", f3d::interactor::ModifierKeys::NONE, "take_screenshot");
+    // "doc"
+    auto docString = [](const std::string& doc) -> std::pair<std::string, std::string>
+    { return std::make_pair(doc, ""); };
+
+    using mod_t = f3d::interaction_bind_t::ModifierKeys;
+    interactor.addBinding({ mod_t::NONE, "Left" }, "load_previous_file_group", "Others",
```

Comment:
Maybe `Others` should just be just an empty string?

Reply:
That would be not great when using `--bindings-list` imo

Reply:
Yes, but I'm thinking of the default argument of `addBinding` if the group is not specified. In this case it makes sense that the group name is empty, and then if needed convert the empty group to `Others` if needed when printing the list.

Reply:
ok fair enough